ui2/src/common/Modal.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Modal } from 'antd';
+import './Button.css';
+
+export const AppModal = ({ children, toggleModal, okDisabled, ...props }) => {
```

Comment:
okDisabled?

Reply:
why not disable?

Reply:
is there 
notOkDisable?

Reply:
In a default modal footer you have a cancel btn and an ok btn, I mean disabling of the ok btn only, should i change to disabled?